### ISSUE TITLE
Running alternative models?

### ISSUE THREAD

**Issue by mr-september** [idosal/AgentLLM#33]
Inspired by an earlier [issue](https://github.com/idosal/AgentLLM/issues/6), the pace of development in this space is overwhelming, and I'm sure this project will gain traction. It wouldn't be feasible for you to manually add support for everything users are requesting then.

Hence, would it be possible to support users choosing their own models, probably locally downloaded as GGML or GPTQ as is popular with [Ooga Booga](https://github.com/oobabooga/text-generation-webui)? This would also allow users flexibilty (i.e. scaling down to 1.3B models or up to 30+B models) according to their devices.

**Comment by idosal** [comment]
Yes, that's something I plan to support (in this and future projects). Stay tuned!

**Comment by Omnikam1111** [comment]
Where in the code is model selection? Can i alter myself to use my Models? Especially for code related tasks where having a specific model for code based questions
